### PR TITLE
do not close prepare

### PR DESCRIPTION
**Pull request by felipeweb**
https://github.com/prest/prest/issues/205

### CODE REVIEW

File: postgres/postgres_test.go [hunk 1]
```diff
@@ -988,6 +988,61 @@ func TestNormalizeGroupFunction(t *testing.T) {
...
+		t.Errorf("expected no errors, but got %v", err)
+	}
+}
+
+func TestCacheInsert(t *testing.T) {
+	sc := Insert("INSERT INTO test(name) VALUES('testcache')")
+	if err := sc.Err(); err != nil {
+		t.Errorf("expected no errors, but got %v", err)
+	}
+	sc = Insert("INSERT INTO test(name) VALUES('testcache')")
```
Comment (marioidival) [inline postgres/postgres_test.go:1018]: This is right? Try save twice same data ?

File: postgres/postgres_test.go [hunk 2]
```diff
@@ -988,6 +988,61 @@ func TestNormalizeGroupFunction(t *testing.T) {
...
+		t.Errorf("expected no errors, but got %v", err)
+	}
+}
+
+func TestCacheDelete(t *testing.T) {
+	sc := Delete("DELETE FROM test WHERE name='test cache'")
+	if err := sc.Err(); err != nil {
+		t.Errorf("expected no errors, but got %v", err)
+	}
+	sc = Delete("DELETE FROM test WHERE name='test cache'")
```
Comment (marioidival) [inline postgres/postgres_test.go:1040]: Return some error here... Try delete same data twice

**Review by marioidival — CHANGES_REQUESTED** [review]

File: postgres/postgres_test.go (hunk 1 above)
Comment (felipeweb) [reply postgres/postgres_test.go:1018]: it is just to test cache

File: postgres/postgres_test.go (hunk 2 above)
Comment (felipeweb) [reply postgres/postgres_test.go:1040]: it is just to test cache

**Comment by felipeweb** [comment]
I have decided to remove the transaction from all the functions because we only execute one operation, there is no need to be under a transaction.
And taking into account that this affected the cache because the commit closed stmt I decided to remove the transaction. 
What do you think about it @avelino @crgimenes @cassiobotaro @marioidival @gtsalles @lerrua ?

**Comment by marioidival** [comment]
@felipeweb First question about it is:
Why do I want cached insert of some specific data?

I aren't SQL specialist but this no make sense to me.

**Comment by felipeweb** [comment]
@marioidival  it is just to test if cache doesn't bug  on insert query i can't imagine this scenario, but in the database is completely possible if table not have unique key

**Review by crgimenes — APPROVED** [review]
LGTM

**Review by lerrua — APPROVED** [review]
:+1:

**Pull request merged**